src/network/networkd-network.h

```diff
@@ -128,6 +128,7 @@ struct Network {
         unsigned dhcp_route_metric;
         uint32_t dhcp_route_table;
         uint16_t dhcp_client_port;
+        bool dhcp_anonymize;
```

Comment:
this commit really needs to come before you add it to the gperf table. I'd probably just merge the two commits into one if I were you, no need to be too finegrained on this thing